changelogs/fragments/641-copy-loadlib-member.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
```

Comment:
My only comment is if this should be a trivial fragment since we have fragment 601 for the code change. Trivial fragments don't get included in the change-log, they are meant to help us, given this is a test cas only, seems like a good idea. 

Here is the one I did for your other PR if you agree to change:
```
trivial:
- zos_copy - Adds a test cases to ensure copying from a PDS/E member containing
  a loadlib to another PDS/E member loadlib member for issue 601.
  (https://github.com/ansible-collections/ibm_zos_core/pull/734)
```

Reply:
I get your point and didn't intend to include a changelog fragment for a new test case, but rather, this is correcting the fact that 601 is the issue number, and by your comments "Format of the fragment name (try to keep it somehow short yet helpful), don’t use the issue number use the PR number thus you will need to have a PR draft before you can get a PR number. [...]" in this PR I removed 601 fragment and added the correct PR # 641 for the fix in the 1.5.0 branch.

Reply:
I am ok with this, i see why we would not want a changelog fragment but I was thinking a bit further down the line, we had discussed a pipeline check to look for changelog fragments.

If we add the check in our pipepline, we won't have a way to easily determine if the PR should have a fragment or not, hence I was just encouraging trivial usage as it won't be part of our changelog but would pass our pipeline in the future.

Reply:
Since there is a fragement now correcting the old one , that is  good, thus I am ok with this now.